packages/react/src/ArqueroTableHeader/ArqueroTableHeader.tsx

```diff
@@ -41,34 +47,35 @@ export const ArqueroTableHeader: React.FC<ArqueroTableHeaderProps> = memo(
 					</H3>
 				) : null}
 				{groupCount ? <H3>{groupCount} groups</H3> : null}
-				{commandItems.length > 0 ? (
-					<CommandBar items={commandItems} styles={commandStyles} />
+				{farCommandItems.length > 0 ? (
+					// Best way to have a command bar in the far right
+					// that handles overflow in case there are too many commands
+					// If the bar is too wide, then only use 10% of it for the commands
+					<CommandBar
+						commands={farCommandItems}
+						width={width >= 992 ? '10%' : undefined}
+					/>
 				) : null}
 			</Header>
 		)
 	},
 )
 
-const commandStyles = {
-	root: {
-		height: HEIGHT,
-		background: 'none',
-		padding: 0,
-	},
-}
-
 const Header = styled.div`
 	height: ${HEIGHT}px;
 	width: 100%;
-	display: flex;
-	justify-content: space-around;
-	align-items: center;
 	background-color: ${({ theme }) => theme.application().accent().hex()};
+	position: relative;
+	padding: 0 5px;
+	box-sizing: border-box;
+	display: inline-flex;
+	align-items: center;
+	justify-content: space-around;
```

Comment:
I think with the two-command-bar options this would now look better with `justify-content: space-between;` to push the command bards to the edges with the metadata centered. That would also require the farCommands to default to flex-end so they are right-justified.